extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -264,7 +263,6 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
     const animationEl = this.win.document.createElement('div');
     animationEl.classList.add('i-amphtml-story-page-attachment-expand');
     const storyEl = closest(this.element, (el) => el.tagName === 'AMP-STORY');
-    this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, false);
```

Comment:
Removed this one because this method is called only from `open_`, which already dispatches the update to the store.